helm/client.go

```diff
@@ -80,11 +79,13 @@ func (c *Client) InstallRelease(releaseName, chart, chartVersion, namespace stri
 	// Check chart dependencies to make sure all are present in /charts
 	chartRequested, err := loader.Load(cp)
 	if err != nil {
+		log.Errorf(err.Error())
```

Comment:
```suggestion
		log.Error(err.Error())
```
https://github.com/sirupsen/logrus/blob/master/exported.go#L213